script/ensure-node-source-fixture-files.js

```diff
@@ -0,0 +1,155 @@
+  sha256sum: '9d55ee072ba6d5a141db092cef1a0f715f7d3fc938285a6d927a1d0a0c7442f7'},
+  {url: `https://nodejs.org/dist/${nodeVersion}/win-x64/node.lib`,
+  filename: 'node_x64.lib',
+  sha256sum: '1bd376a23d181d85096d1a9c46e6be7fcd20d30f9b8f77a2a847d3dbff8e25c7'},
+  {url: `https://nodejs.org/dist/${nodeVersion}/win-x86/node.lib`,
+  filename: 'node.lib',
+  sha256sum: 'b1c6dc670911d85ef1704fa56f4cc4c7e1071f4869778398e6d88b3b0b565978'},
+  {url: `https://nodejs.org/dist/${nodeVersion}/SHASUMS256.txt`,
+  filename: 'SHASUMS256.txt',
+  sha256sum: '64aad1211a6003dd6529ebf9f19167769d0356ce5affc4245bb26c35aa66a9ed'},
+] // If you ever need to update to a newer Node version, just calculate the sha256sums
+  // for the new files and update them here to match the new files.
+
+const sourceFixtureDir = path.resolve(__dirname, '..', 'spec', 'fixtures', 'node-source');
+fs.mkdirSync(sourceFixtureDir, {recursive: true});
+
+for (const details of filesToFetch) {
+  ensureFile(details);
+} // This is the main loop, with support functions below.
+
+async function ensureFile (details) {
+  // We check if the file already exists, and if so,
+  // make sure its sha256sum matches the expected/correct value.
+  // If it doesn't exist or has the wrong hash, try to re-download.
+
+  logVerbose('details is:');
+  logVerbose(details);
+
+  const destinationPath = path.resolve(sourceFixtureDir, details.filename);
+  logVerbose('destinationPath is: ' + destinationPath);
+
+  const existingFileIsCorrect = await verifyExistingFile(destinationPath, details.sha256sum);
+
+  if (!existingFileIsCorrect) {
+    logVerbose('Hash did not match, re-downloading...')
+    // Get the file
+    downloadFileToDestination(details.url, destinationPath, details.sha256sum)
+      .then(async function () {
+        console.log(`Successfully downloaded file from ${details.url}.`)
+        logVerbose('checking if hash matches in for...of loop.');
+        // Check its hash
+        const hashDidMatch = await verifyHash(destinationPath, details.sha256sum);
+        if (!hashDidMatch) {
+          console.error(`Hash did not match for ${destinationPath}`);
+        }
+      })
+      .catch(console.error);
+  }
+}
+
+async function verifyExistingFile (targetPath, expectedHash) {
+  if (fs.existsSync(targetPath)) {
+    logVerbose(targetPath + " already exists.");
+
+    logVerbose(`verifying hash for ${targetPath} in verifyExistingFile`)
+    const hashDidMatch = await verifyHash(targetPath, expectedHash)
+
+    if (hashDidMatch) {
+      // Successfully verified existing file. Return true.
+      console.log(`Existing file ${targetPath} successfully verified.`)
+      return true;
+    } else {
+      // Existing file's hash was wrong. Delete the old file.
+      console.error(`Hash did not match for ${targetPath}. Deleting.`);
+      fs.rmSync(targetPath);
+    }
+  } else {
+    // File did not actually exist.
+    logVerbose(targetPath + " does not exist.");
```

Comment:
```suggestion
    logVerbose(`${targetPath} does not exist.`);
```
Personal Suggestion: Template strings are always much nicer than concatenating